build/Common.prod.props

```diff
@@ -34,10 +34,6 @@
     </PropertyGroup>
   </Target>
 
-  <ItemGroup>
```

Comment:
Both `Common.prod.pros` and `Common.test.props` have this, so I moved it to `Common.props`.